pkg/psmdb/backup/pbm.go

```diff
@@ -933,8 +952,111 @@ func (b *pbmC) GetBackupMeta(ctx context.Context, bcpName string) (*backup.Backu
 	return backup.NewDBManager(b.Client).GetBackupByName(ctx, bcpName)
 }
 
+// deleteBackup deletes backup with the given name from the current storage and pbm database
+// deleteBackup, deleteBackupImpl and deleteIncremetalChainImpl is copied from PBM v2.11.0 to fix PBM-1633
+// we need to stop maintaining these in operator v1.24.0
+func deleteBackup(ctx context.Context, conn connect.Client, name, node string, event pbmLog.LogEvent) error {
+	bcp, err := backup.NewDBManager(conn).GetBackupByName(ctx, name)
+	if err != nil {
+		return errors.Wrap(err, "get backup meta")
+	}
+
+	if bcp.Type == defs.IncrementalBackup {
+		return deleteIncremetalChainImpl(ctx, conn, bcp, node, event)
```

Comment:
Typo: incremental instead of incremetal 🤘🏽

Reply:
pbm code, i don't want to change it